chain/chunks/src/lib.rs

```diff
@@ -1416,6 +1423,82 @@ mod test {
         assert!(network_adapter.requests.read().unwrap().is_empty());
     }
 
+    #[cfg(feature = "expensive_tests")]
+    #[test]
+    fn test_seal_removal() {
```

Comment:
Do we need an expensive test for this? Seems like a unit test would accomplish it just as well.